krabs/krabs/testing/event_filter_proxy.hpp

```diff
@@ -71,4 +71,27 @@ namespace krabs { namespace testing {
         event_filter_.on_event(record);
     }
 
+    class threadsafe_event_filter_proxy {
+    public:
+        threadsafe_event_filter_proxy(krabs::threadsafe_event_filter& filter);
+        void push_event(const synth_record& record);
```

Comment:
[nit] The rest of the project has & and * on the identifier, not the type - please update to be consistent.